features/git-extract/cherry_pick_conflict/without_remote_origin/with_open_changes.feature

```diff
@@ -0,0 +1,101 @@
+Feature: git extract: resolving conflicts between main branch and extracted commits (with open changes and without remote origin)
+
+  As a developer extracting a commit that conflicts with the main branch and without a remote repository
+  I want to be given the choice to resolve the conflicts or abort
+  So that I can finish the operation as planned or postpone it to a better time.
```

Comment:
This shouldn't be necessary but should instead just reference another user value. Probably
```
(see ../with_remote_origin/with_open_changes.feature)
```